README.md

```diff
@@ -1,84 +1,55 @@
-# razorpay-php
+<p align="center"><img src="https://razorpay.com/images/logo.png"></p>
+<p align="center">
+<a href="https://travis-ci.org/razorpay/razorpay-php"><img src="https://travis-ci.org/razorpay/razorpay-php.svg?branch=master"></a>
+<a href="https://packagist.org/packages/razorpay/razorpay"><img src="https://poser.pugx.org/razorpay/razorpay/v/stable.svg"></a>
+<a href="https://github.com/razorpay/razorpay-php/blob/master/LICENSE.txt"><img src="https://poser.pugx.org/razorpay/razorpay/license.svg"></a>
+</p>
 
-[![Build Status](https://travis-ci.org/razorpay/razorpay-php.svg?branch=master)](https://travis-ci.org/razorpay/razorpay-php) [![Latest Stable Version](https://poser.pugx.org/razorpay/razorpay/v/stable.svg)](https://packagist.org/packages/razorpay/razorpay) [![License](https://poser.pugx.org/razorpay/razorpay/license.svg)](https://packagist.org/packages/razorpay/razorpay)
+## Introduction
 
-Razorpay client PHP Api. The api follows the following practices:
+The Razorpay PHP package makes the task of integrating the Razorpay payment gateway dead simple in your PHP project by providing an elegant & well tested wrapper around the Razorpay API making integration with PHP projects more neat & simple. We don't want to overwhelm you but just a few prerequisite sort of stuff we want you to know are as follows. They are really very few _promise!_ 
 
-- namespaced under Razorpay\Api
-- call $api->class->function() to access the api
-- api throws exceptions instead of returning errors
-- options are passed as an array instead of multiple arguments wherever possible
+- API throws exceptions instead of returning errors
+- Options are passed as an array instead of multiple arguments wherever possible
 - All request and responses are communicated over JSON
 - A minimum of PHP 5.3 is required
 
-# Installation
+## Installation
 
-- If your project uses composer, add following to composer.json
-```json
-{
-    "require": {
-        "razorpay/razorpay": "1.*"
-    }
-}
-```
+- **Via Composer**
 
-Then, run `composer update`. If you are not using composer, download
-the latest release from [the releases section](https://github.com/razorpay/razorpay-php/releases).
-**You should download the `razorpay-php.zip` file**.
```

Comment:
We can mark composer as the preferred way. However, the reality is that even frameworks like CodeIgniter don't support composer natively. This single line has helped us reduce support lots of support queries, and we can't drop it.